docs/source/scancode-toolkit/cli-reference/core-options.rst

```diff
@@ -0,0 +1,282 @@
+`Core` Options
+==============
+
+.. include::  /scancode-toolkit/rst_snippets/basic_options.rst
+
+.. _cli_core:
+
+.. include::  /scancode-toolkit/rst_snippets/core_options.rst
+
+``--generated`` Options
+-----------------------
+
+The ``--generated`` option classifies automatically generated code files with a flag.
+
+An example of using ``--generated`` in a scan::
+
+    ./scancode -clpieu --json-pp sample_generated.json samples --generated
+
+In the results, for each file the following attribute is added with it's corresponding True/False
+value ::
+
+    "is_generated": true
+
+For the samples folder, the following files have a true value for their is_generated attribute::
+
+    "samples/zlib/dotzlib/LICENSE_1_0.txt"
+    "samples/JGroups/licenses/apache-2.0.txt"
+
+..
+    [ToDo] Research and Write Better
+
+``--max-email`` Options
```

Comment:
Should not this be side-by-side with the --email option? (same for the --url option below?)

Reply:
I've followed the groupings and order of the options in the help-text as users can easily switch between both without having to search. The present grouping (and in the help-text) is, in my opinion, the most logical one. What can be done is adding Notes to `--max-email` at `-e` and the same for the email. I'll do that.

Reply:
This issue has been fixed.